### PR TITLE
Fixed location of authoritative repositories

### PR DESCRIPTION
**Pull request by ligiabernardet**
Fixed location of authoritative repositories. Thank you @junwang-noaa for bringing this up.

### CODE REVIEW

**Comment by climbfuji** [comment]
@ligiabernardet since you are already creating the PR for ufs-weather-model, you might as well do the one for fv3atm with the updated submodule pointers for ccpp-physics and ccpp-framework, and do the same for the updated submodule pointer for fv3atm here.

**Comment by ligiabernardet** [comment]
I can do this. It will be my first time with nested PRs, so we'll see if I
can manage!!

**Comment by climbfuji** [comment]
I can help a bit.

You can start from your existing PR for the ufs-weather-model and check out the branches that correspond to the PRs for ccpp-framework and ccpp-physics inside fv3atm. A git status will then show you that something has changed in fv3atm. Edit .gitmodules, commend out the existing url= and branch= entries for ccpp-physics and ccpp-framework, add the temporary urls and branches instead. Then commit the change to .gitmodules and the updated submodule pointers for ccpp-framework and ccpp-physics with a commit message saying sth like "Update .gitmodules and submodule pointers for ccpp-framework and ccpp-physics for code review and testing", create a local branch, push it to GitHub and create the fv3atm PR. Then do the same for fv3atm in ufs-weather-model in your existing PR.

**Comment by ligiabernardet** [comment]
I updated the submodules. Please see if this worked as it should. Tks.
Associated PRs
https://github.com/NOAA-EMC/fv3atm/pull/90
https://github.com/NCAR/ccpp-physics/pull/420
https://github.com/NCAR/ccpp-framework/pull/274

File: doc/UsersGuide/source/CodeOverview.rst [hunk 1]
```diff
@@ -45,12 +45,12 @@ The umbrella repository for the UFS Weather Model is named ufs-weather-model.  U
 
    ufs-weather-model/
    ├── FMS                                     https://github.com/NOAA-GFDL/FMS
-   ├── FV3                                     https://github.com/NCAR/fv3atm
-   │   ├── atmos_cubed_sphere                  https://github.com/NCAR/GFDL_atmos_cubed_sphere
+   ├── FV3                                     https://github.com/NOAA-EMC/fv3atm
+   │   ├── atmos_cubed_sphere                  https://github.com/NOAA-EMC/GFDL_atmos_cubed_sphere
```
Comment (junwang-noaa) [inline doc/UsersGuide/source/CodeOverview.rst]: GFDL_atmos_cubed_shere in public release is pointing to NOAA-GFDL repo.

Comment (ligiabernardet) [reply doc/UsersGuide/source/CodeOverview.rst]: Tks This has been addressed.

Comment (junwang-noaa) [reply doc/UsersGuide/source/CodeOverview.rst]: I see line 36 is changed, but line 49 is still listing NOAA-EMC, not NOAA-GFDL.

**Review by junwang-noaa — APPROVED** [review]

**Comment by climbfuji** [comment]
Thanks, Jun. I am going to approve and merge the ccpp-framework and ccpp-physics PRs, then I will ask Ligia to update the submodule pointers in the fv3atm and revert the .gitmodules change.

Oh, after running the tests of course.

**Comment by climbfuji** [comment]
Regression tests passed on hera.intel, cheyenne.intel, cheyenne.gnu. Will start merging the CCPP PRs.

**Comment by climbfuji** [comment]
@ligiabernardet The CCPP PRs have been merged. Now you need to use your local working directory from which you created the PRs.
```
cd FV3/ccpp/framework
git remote update
# assuming that the remote 'origin' points to the NCAR repository
git checkout origin/release/public-v4
cd ../physics
git remote update
# assuming that the remote 'origin' points to the NCAR repository
git checkout origin/release/public-v4
cd ../..
# now you are in directory FV3 and a 'git status' should show you that there are new commits for ccpp/framework and ccpp/physics
```
Then edit `.gitmodules` in FV3 and revert the change you made earlier. Then:
```
git add .gitmodules ccpp/framework ccpp/physics
git commit -m "Revert change to .gitmodules and update submodule pointers for ccpp-framework and ccpp-physics"
git push WHATEVER_YOUR_REMOTE_NAME_IS_LOCALLY doc_updates
```
We can check if the changes/hashes are ok and Jun can merge the fv3atm PR.

**Comment by climbfuji** [comment]
@ligiabernardet Now that the fv3atm has been merged you need to do the same for the ufs-weather-model PR. The correct has for fv3atm (i.e. `git log` in subdirectory FV3) is (the name `noaa-emc/release/public-v1` could be different, depending on how you called the NOAA-EMC remote destination locally, I called it noaa-emc):
```
commit 3b2671b554c3f0a06a369019ba6901f66c6fcb6a (HEAD, noaa-emc/release/public-v1)
Author: ligiabernardet <ligia.bernardet@noaa.gov>
Date:   Tue Mar 31 19:55:55 2020 -0600

    Doc update for public release (submodules ccpp-physics and ccpp-framework) (#90)

    * Update .gitmodules and submodule pointers for ccpp-framework and ccpp-physics for code review and testing

    * Revert change to .gitmodules and update submodule pointers for ccpp-framework and ccpp-physics

    * Update submodule pointers for CCPP
```

**Comment by ligiabernardet** [comment]
The FV3 submodule has been updated.

**Comment by climbfuji** [comment]
This looks good to me, thanks. @junwang-noaa @DusanJovic-NOAA whenever it is convenient for you ...

**Pull request merged**